ext/mysqli/mysqli_nonapi.c

```diff
@@ -673,12 +673,11 @@ static int mysqlnd_zval_array_to_mysqlnd_array(zval *in_array, MYSQLND ***out_ar
 /* }}} */
 
 /* {{{ mysqlnd_zval_array_from_mysqlnd_array */
-static int mysqlnd_zval_array_from_mysqlnd_array(MYSQLND **in_array, zval *out_array)
+static zend_result mysqlnd_zval_array_from_mysqlnd_array(MYSQLND **in_array, zval *out_array)
```

Comment:
The result value of `mysqlnd_zval_array_from_mysqlnd_array` is never read, should we also drop it?

Reply:
Probably, or start using the return value.